be/src/storage/rowset/segment_writer.h

```diff
@@ -56,6 +56,7 @@ struct SegmentWriterOptions {
     uint32_t storage_format_version = 1;
     uint32_t num_rows_per_block = 1024;
     vectorized::GlobalDictByNameMaps* global_dicts = nullptr;
+    std::vector<std::size_t> column_indexes;
```

Comment:
```suggestion
    std::vector<uint32_t> column_indexes;
```
I think uint32_t is enough